packages/desktop/src/renderer/app/services/data.service.ts

```diff
@@ -163,6 +163,37 @@ export class DataService extends Logger {
     return databucket;
   }
 
+  /**
+   * Renew one databucket ID
+   *
+   * @param params
+   */
+  public renewDatabucketID(databucket: DataBucket) {
+    databucket.id = (Math.random() + 1).toString(36).substring(2, 6);
```

Comment:
dedup string generation (between schema and here) and put in a utils (in commons)